storage/innobase/include/ut0ut.h

```diff
@@ -185,7 +188,8 @@ Sprintfs a timestamp to a buffer, 13..14 chars plus terminating NUL. */
 void
 ut_sprintf_timestamp(
 /*=================*/
-	char*	buf); /*!< in: buffer where to sprintf */
+	char*	buf); /*!< in: buffer where to sprintf. Must be at least
+			`UT_BUF_MIN_LEN` long. */
```

Comment:
This should be `UT_` ***`TS_`*** `BUF_MIN_LEN`

Reply:
Good catch, updated this